src/platform/silabs/efr32/BLEManagerImpl.cpp

```diff
@@ -586,18 +588,26 @@ CHIP_ERROR BLEManagerImpl::StartAdvertising(void)
     SuccessOrExit(err);
 
     sl_bt_advertiser_configure(advertising_set_handle, 1);
+
     ret = sl_bt_legacy_advertiser_start(advertising_set_handle, connectableAdv);
+    err = MapBLEError(ret);
+    SuccessOrExit(err);
 
-    if (SL_STATUS_OK == ret)
+    if (mFlags.Has(Flags::kFastAdvertisingEnabled))
     {
-        if (mFlags.Has(Flags::kFastAdvertisingEnabled))
-        {
-            StartBleAdvTimeoutTimer(CHIP_DEVICE_CONFIG_BLE_ADVERTISING_INTERVAL_CHANGE_TIME);
-        }
-        mFlags.Set(Flags::kAdvertising);
+        StartBleAdvTimeoutTimer(CHIP_DEVICE_CONFIG_BLE_ADVERTISING_INTERVAL_CHANGE_TIME);
     }
+    mFlags.Set(Flags::kAdvertising);
 
-    err = MapBLEError(ret);
+    if (postAdvChangeEvent)
+    {
+        // Post CHIPoBLEAdvertisingChange event.
+        ChipDeviceEvent advChange;
+        advChange.Type                             = DeviceEventType::kCHIPoBLEAdvertisingChange;
```

Comment:
Change looks reasonable, but ... isn't CHIPoBLE a BLE operational network? We don't support that. So is this just a weirdly named event?

Reply:
I guess so... Event was named 4 years ago based on the blame. I can create a follow up ticket to remove the `CHIPo` part from the name event name.

Reply:
okie dokie.